packages/devtools_app/lib/src/network/http.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// This library contains code pulled from dart:io that is not platform specific.
```

Comment:
If this works on web, it's a shame we can't just import it from dart:io.

Reply:
@ditman who is interested in answering just that sort of question. :)

Reply:
Yup, there's no platform specific code in a large portion of the `dart:_http` code :-(

Reply:
This is great, and I don't want to derail this PR, but maybe some of the base work here (Cookie, Http Headers...) may have a better home in [package:http](https://pub.dev/packages/http)?

_(This is something that can be extracted/refactored later)_